binding.c

```diff
@@ -6,8 +6,19 @@
 
 typedef struct {
   js_context_t *context;
+  bool is_destroyed;
 } bare_realm_t;
 
+static void
+on_realm_finalize (js_env_t *env, void *data, void *finalize_hint) {
```

Comment:
```suggestion
bare_realm__on_finalize (js_env_t *env, void *data, void *finalize_hint) {
```

Keeping all symbols prefixed helps with debugging so you can immediately spot where a symbol comes from. The double `__` indicates that it's an internal symbol.